packages/oc-template-jade-compiler/to-be-published/hash-builder.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
We may want to investigate https://github.com/puleos/object-hash
It is widely used, dep free, well maintained, apparently, and defaults to sha1+hex.